extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -378,6 +378,11 @@ private function tarFiles( $directory, $dbname, $cityid ) {
 			$this->log( "List of files in {$directory} is empty" );
 			throw new WikiaException( "List of files in {$directory} is empty" );
 		}
+
+		// SUS-4325 | CloseWikiMaintenance should remove directories with images after tar file is created
+		$this->log( "Removing '{$directory}' directory" );
```

Comment:
from `Tar->create()` `The method return false and a PEAR error text`
please Justin Case™  check is $result false

Reply:
Good point - see fb286a6